asset/quickstart/list-saved-queries/maiin_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package list
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	asset "cloud.google.com/go/asset/apiv1"
+	"cloud.google.com/go/asset/apiv1/assetpb"
+	cloudresourcemanager "google.golang.org/api/cloudresourcemanager/v1"
+)
+
+var (
+	projectID     string
+	savedQueryID  string
+	projectNumber string
+
+	ctx    context.Context
+	client *asset.Client
+)
+
+func TestMain(m *testing.M) {
```

Comment:
In Go, `TestMain` is used for setup.  

You'll want to have a `TestListSavedQueries(t *testing.T)` to test your sample

Reply:
the TestListSavedQueries function is in the lower section of this file:)